src/main/java/de/dataelementhub/rest/controller/v1/ElementController.java

```diff
@@ -355,4 +355,30 @@ public ResponseEntity getMembers(
       return new ResponseEntity<>(HttpStatus.NOT_FOUND);
     }
   }
+
+  /**
+   * Update dataElementGroup or record members.
+   * If changes acquired return the new location otherwise return the old one.
+   */
+  @PutMapping("/{urn}/members")
+  @Order(SecurityProperties.BASIC_AUTH_ORDER)
+  public ResponseEntity updateMembers(@PathVariable(value = "urn") String urn,
+      UriComponentsBuilder uriComponentsBuilder) {
+    try {
+      String newUrn = elementService.updateMembers(
+          DataElementHubRestApplication.getCurrentUser().getId(), urn);
+      UriComponents uriComponents;
+      uriComponents =
+          uriComponentsBuilder.path("/v1/element/{urn}").buildAndExpand(newUrn);
```

Comment:
Not particularly a problem - and surely done earlier by someone (probably me :( ) - but we should make at least the "v1" part a constant in the v1 element controller to avoid mistakes later on